kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -66,12 +76,12 @@ def build(self):
         self.pad_left, self.pad_right = compute_padding(self.J_pad, self.N)
         # compute start and end indices
         self.ind_start, self.ind_end = compute_border_indices(
-            self.J, self.pad_left, self.pad_left + self.N)
+            self.log2_T, self.pad_left, 2**self.J_pad - self.pad_right)
```

Comment:
I'm not following the last argument here. Was this a bug previously?

Reply:
Existing logic fails with padding by more than `N`.

Reply:
> Looks like this is resolved?

Old version is highlighted, see [latest](https://github.com/kymatio/kymatio/blob/1ee37e84ac3c0050440f64910a395fab97a30bf8/kymatio/scattering1d/frontend/base_frontend.py#L58).

Reply:
I just confirmed they're 100% identical (aside for `log2_T`) - my confusion, will remove.

Reply:
Ok. Resolving.